src/icons/_settings.scss

```diff
@@ -606,8 +606,8 @@ $fd-icons: (
   circle-task-2: "\e255",
   project-definition-triangle: "\e256",
   project-definition-triangle-2: "\e257",
-  master-task-triangle: "\e258",
-  master-task-triangle-2: "\e259",
+  main-task-triangle: "\e258",
```

Comment:
this is not own by us. We should raise it to SAP UX as they own the icons.
We can remove the support

Reply:
Didn't mean to stage these